Graph and Linked List Practice Game/Assets/Scripts/Graph.cs

```diff
@@ -0,0 +1,202 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEditor.U2D.Common;
+using UnityEngine.UI;
+
+namespace GraphSearching
+{
+    //This is the graph class
+    public class Graph : MonoBehaviour
```

Comment:
We'll need to figure out how to decouple this from Monobehavior, but that'll require significant refactoring, so it's okay for now